### DISCUSSION TITLE
Consistent loading...

### DISCUSSION THREAD

**Discussion by zuihou** [minio/minio#16613] [Q&A] [answered]
## Expected Behavior
Normal access

## Current Behavior
Access “object browser” ，view files in a bucket and keep loading...
 
## Steps to Reproduce (for bugs)
  
<img width="1828" alt="image" src="https://user-images.githubusercontent.com/8500410/218624714-7d1a132b-d0e6-43bb-929e-6d76efb35641.png">

## Your Environment

* Version used (`minio --version`):
```
minio version RELEASE.2023-01-25T00-19-54Z (commit-id=50d58e9b2dbbfbe2f3afa2c16ae701f9c1cf03ec)
Runtime: go1.19.4 linux/amd64
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Copyright: 2015-2023 MinIO, Inc.
```
* Server setup and configuration:
```

nohup /home/pjjs/minio/minio server --config-dir /home/pjjs/minio/config /home/pjjs/minio_data --address 172.26.3.90:9000 --console-address 172.26.3.90:9090  > /home/pjjs/minio/minio.log 2>&1 &
```
* Operating System and version (`uname -a`):
```
Linux localhost.localdomain 4.18.0-240.15.1.el8_3.x86_64 #1 SMP Mon Mar 1 17:16:16 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by harshavardhana** [comment, marked as answer]
If you have millions of objects dumped under a prefix without hierarchy the access may take time. There is nothing special here.

**Reply by zuihou** [reply]
no object.

**Reply by harshavardhana** [reply]
Then is your load balancer supporting websockets ?

**Reply by zuihou** [reply]
After changing the browser, the problem was solved. .........

**Reply by Haoke98** [reply]
I also faced the problem like that, and I found that it is usually appears on the Chrome.

**Reply by dianshane** [reply]
Dear @Haoke98,
i haven't worked on any minio project since the one that lead me to this thread.

I cannot 100 percent recall the exact solution but i am pretty confident that it had to do with the way sockets were being setup between the client, my reverse proxy and my minio containers.

Though i cannot exactly recall the fix, i still have all the final working configuration files(and possibly some history from old commits if you are interested in seeing how i got it fixed), under gpl2. 

If you so wish i can very well upload them for you in a repo. Please let me know.

Sotiris from Greece

**Comment by dianshane** [comment]
i seem to have the same issue here, tried everything, changed browsers, cleared cache checked for ram and cpu
any idea is welcome.

**Reply by bralbral** [reply]
Is Your minio instalation behind proxy? If yes , you need turn on http 1.1 for ws.
